view/partial/task-header.html

```diff
@@ -24,10 +24,10 @@ <h1 class="h2 crunch-top">{{task.name}}</h1>
 			<div class="run-details task-header clearfix">
 				<div class="col-md-12 clearfix">
 					{{#unless readonly}}
-				        <ul class="inline-list" role="menu">
-				          	<li><a href="/{{task.id}}/edit">Edit this task</a></li>
-							<li><a href="/{{task.id}}/delete">Delete this task</a></li>
-							<li><a href="{{task.hrefRun}}" data-test="run-task">Run Pa11y</a></li>
+						<ul class="inline-list" role="menu">
```

Comment:
Adding `role="presentation"` to the list items and `role="menu"` to the links _does_ solve the problem of this list not being marked up correctly as a menu... but I think the real problem here is not that it's marked up incorrectly as a menu, but more that the `role="menu"` is even there in the first place. 

The ARIA specification is specific about when a `menu` or `menubar` role is expected and what it's for (my emphasis):
>A menu is a widget that offers a list of choices to the user, such as a set of actions or functions. **Menu widgets behave like native operating system menus, such as the menus that pull down from the menubars commonly found at the top of many desktop application windows.** 
https://www.w3.org/TR/wai-aria-practices-1.2/#menu

The ARIA spec _does_ give a suggestion for using the `menu` role for a navigation, but it's absurdly complex with multiple levels of menus, and the one we have on Pa11y Dashboard is just three links to common actions. The `menu` pattern is overkill here. IMO this would be better marked up as a ul with a `<nav aria-label="task tools">` wrapped around it. 

Here's a blog post by Adrian Roselli on the subject that I like: https://adrianroselli.com/2017/10/dont-use-aria-menu-roles-for-site-nav.html